src/applications/mhv-secure-messaging/components/ComposeForm/RecipientsSelect.jsx

```diff
@@ -213,22 +217,41 @@ const RecipientsSelect = ({
 
   return (
     <>
-      <VaSelect
-        enable-analytics
-        id="recipient-dropdown"
-        label="To"
-        name="to"
-        value={defaultValue !== undefined ? defaultValue : ''}
-        onVaSelect={handleRecipientSelect}
-        class="composeSelect"
-        data-testid="compose-recipient-select"
-        error={error}
-        data-dd-privacy="mask"
-        data-dd-action-name="Compose Recipient Dropdown List"
-      >
-        <CantFindYourTeam />
-        {optionsValues}
-      </VaSelect>
+      {isComboBoxEnabled ? (
+        <VaComboBox
+          required
+          id="recipient-combobox"
```

Comment:
if this `id` attribute is not used for any styling or functional purposes, we dont need to add it. For testing, we can use `data-testid`

Reply:
Thank you.  I added it originally to try to override the width, but that didn't work and I forgot to remove it.  I'll take it out